src/ActivityForm.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import "./ActivityForm.css";
+
+export default function ActivityForm({ onAddActivity }) {
+  //console.log(onAddActivity);
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const newActivity = {
+      name: event.target.elements.activity.value,
+      isForGoodWeather: event.target.elements.weatherActivity.checked,
+    };
+
+    onAddActivity(newActivity);
+    console.log("ActivityForm: ", newActivity);
+
+    event.target.reset();
+    event.target.elements.activity.focus();
+  }
+
+  return (
+    <div>
+      <h1>Add new Activity</h1>
+      <form onSubmit={handleSubmit}>
+        <label> Add New Activity: </label>
+        <section className="beside">
+          <label>Name: </label>
+          <input
+            id="activity"
+            type="text"
+            name="activity"
+            placeholder="Walking"
+          />
```

Comment:
You are creating a form input field for a name, but there are a couple of important elements missing for it to function optimally:

```suggestion
          <label htmlFor="activity">Name: </label>
         <input id="activity" type="text" name="activity" placeholder="Walking" />
```